### PR TITLE
Fix button alignment - fixes #25

### LINKED ISSUE #25: New Github styles moves New Pull Request button
Github has a (new) new style and now things are wonky again.

image

### PR DESCRIPTION
**Pull request by clayreimann**
I wish you could `git push --force` with the Github GUI :cry:

### CODE REVIEW

**Comment by sindresorhus** [comment]
> with the Github GUI

Please don't use the GitHub UI for actual code changes.

File: content.js [hunk 1]
```diff
@@ -81,7 +81,12 @@ function trigger() {
 	toggleFiles();
 }
 
+function adjustOtherButtons() {
+	document.getElementsByClassName('new-pull-request-btn')[0].textContent = 'New PR';
+}
+
 document.addEventListener('DOMContentLoaded', () => {
+	adjustOtherButtons();
```
Comment (sindresorhus) [inline content.js]: This doesn't work. Needs to be added in the `trigger` function.

Comment (clayreimann) [inline content.js]: It works fine for me. What about it doesn't work?

**Comment by clayreimann** [comment]
> >with the Github GUI
> Please don't use the GitHub UI for actual code changes.

I meant the desktop client, in any case I fixed the git remote to use ssh. `push --force` enabled!

File: content.js (hunk 1 above)
Comment (sindresorhus) [inline content.js]: Go to a repo, click the "Pull Request" tab, then click the "Code" tab, and you'll see the new PR button says "New pull request" again.

**Comment by radiovisual** [comment]
I have confirmed that this PR does what it claims to do, and it works, so it's safe for a merge, however, I just wanted to point out that I think I the "New Pull Request" doesn't need to be changed. There is plenty of room for the "New Pull Request" button to fit on the existing UI without modifying it. This PR makes the  button smaller, and changes the text to "New PR". If you compare the unchanged button with the changed button, my vote is with the button being **unchanged**:

before-after

These images above are screenshots from my browser, where I pulled in @clayreimann's working branch, so I know that the button has enough room to fit and still work as expected.

**Comment by sindresorhus** [comment]
Thanks @clayreimann :) Sorry about the delay in merging.

**Comment by sindresorhus** [comment]
@radiovisual That doesn't work when the branch is not named `master`:

screen shot 2016-03-15 at 17 24 05

**Comment by radiovisual** [comment]
Ok, now I see. Thanks.

**Pull request closed without merging**